BREAKING_CHANGES.md

```diff
@@ -2,6 +2,61 @@
 
 ## v2.0.0
 
+### `ix-flip-tile`
+
+Change flip-state from
+
+```ts
+export enum FlipTileState {
+  None = 'none',
+  Info = 'info',
+  Warning = 'warning',
+  Alarm = 'alarm',
+  Primary = 'primary',
+}
+```
+to
+
+```ts
+export enum FlipTileState {
+  None = 'None',
+  Info = 'Info',
+  Warning = 'Warning',
+  Alarm = 'Alarm',
+  Primary = 'Primary',
+}
+```
```

Comment:
Should be the other way around (from None to none)?

Reply:
Ja

Reply:
Fixed :)